pontoon/base/views.py

```diff
@@ -1064,9 +1064,8 @@ def amagama(request):
         return HttpResponse("error")
 
 
-def transvision(request, repo, title):
+def transvision(request):
     """Get Mozilla translations from Transvision service."""
-    log.debug("Get Mozilla translations from Transvision service.")
```

Comment:
Why do we need this view? Maybe there's a way to move this thing from backend to client-side (i'm not sure what kind of apis transvision provides).